src/base/Context.ts

```diff
@@ -575,6 +583,30 @@ export class Context {
     return id;
   }
 
+  /**
+   * @hidden
+   *
+   * Returns an Child Identity when given a DID string
+   *
+   * @throws if the Child Identity does not exist
+   */
+  public async getChildIdentity(child: ChildIdentity | string): Promise<ChildIdentity> {
+    if (child instanceof ChildIdentity) {
+      return child;
+    }
+    const id = new ChildIdentity({ did: child }, this);
```

Comment:
```suggestion
    const childIdentity = new ChildIdentity({ did: child }, this);
```
or `childId`

Just `id` made me think its a number at first